src/js/common/components/Widgets/ReadMore.jsx

```diff
@@ -162,6 +202,9 @@ ReadMore.propTypes = {
   numberOfLines: PropTypes.number,
   onShowMoreAlternateFunction: PropTypes.func,
   textToDisplay: PropTypes.node.isRequired,
+  eventName: PropTypes.string,
+  politicianIdShowMore: PropTypes.string,
+  politicianNameShowMore: PropTypes.string,
```

Comment:
You can remove politicianNameShowMore since we will get Politician data from the PoliticianStore. Please change politicianIdShowMore to `politicianWeVoteId`